engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -806,6 +807,7 @@ namespace dmGameSystem
     {
         GuiComponent* gui_component = (GuiComponent*)*params.m_UserData;
         dmGui::Result result = dmGui::FinalScene(gui_component->m_Scene);
+        dmGui::ForceRemoveDynamicTextures(gui_component->m_Scene, &DeleteTexture);
```

Comment:
I thought about it and not sure if we should do it in `CompGuiOnReload` as well?

Reply:
Yes, seems reasonable to also do it in reload.

Reply:
It makes sense. Dine.